### PR TITLE
separate dbt checks test cases

### PR DESCRIPTION
**Pull request by johannkm**
less confusing than testing two things in one case

### CODE REVIEW

**Comment by johannkm** [comment]
Current dependencies on/for this PR:
* master
  * **PR #17954** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/17954?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/17954?utm_source=stack-comment).

File: python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py [hunk 1]
```diff
@@ -30,6 +30,19 @@
 )
 
 
+@pytest.fixture(params=[[["build"]], [["seed"], ["run"], ["test"]]])
+def dbt_commands(request):
+    return request.param
+
+
+@pytest.fixture()
+def mocked_context(mocker: MockerFixture):
```
Comment (rexledesma) [inline python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py:39]: Probably doesn't need to be a fixture if it's used only in one place.

File: python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py [hunk 2]
```diff
@@ -170,14 +166,18 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
     assert result.success
 
+
+def test_asset_checks_are_logged(mocked_context, dbt_commands: List[List[str]]):
```
Comment (rexledesma) [inline python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py:170]:
```suggestion
def test_asset_checks_are_logged(mocked_context: MagicMock, dbt_commands: List[List[str]]):
```

Comment (rexledesma) [inline python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_checks.py:170]: Something to suggest that this is a resource level test

```suggestion
def test_asset_checks_are_logged_from_resource(mocked_context, dbt_commands: List[List[str]]):
```

**Review by rexledesma — APPROVED** [review]

**Pull request merged**